packages/common/src/types/savedCharts.ts

```diff
@@ -52,6 +54,15 @@ export type BigNumberConfig = {
     config?: BigNumber;
 };
 
+export type PieChart = {
+    isDonut?: boolean;
```

Comment:
I guess this will be getting more configuration on how the fields map to the pie chart? I guess you can only choose one metric for the values, and the dimensions will automatically generate the labels?

Reply:
yep, this will have more config options. there's a whole milestone for it https://github.com/lightdash/lightdash/milestone/99